assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -123,6 +123,31 @@ export const controls = {
 };
 
 export const selectors = {
+	/**
+	 * Checks whether changes are currently loading for a module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string} slug Slug for module store.
+	 * @return {boolean} Whether or not changes are currently loading for the module.
+	 */
+	areSettingsEditDependenciesLoaded: createRegistrySelector(
+		( select ) => ( state, slug ) => {
+			invariant( slug, 'slug is required.' );
+			const storeName = select( CORE_MODULES ).getModuleStoreName( slug );
+			const moduleStore = select( storeName );
+			if ( ! moduleStore ) {
+				return undefined;
+			}
+			const areSettingsEditDependenciesLoaded =
+				moduleStore.areSettingsEditDependenciesLoaded?.();
+			return (
+				areSettingsEditDependenciesLoaded ||
+				areSettingsEditDependenciesLoaded === undefined
+			);
```

Comment:
I think we can more/less inline this to make it a bit more clear as to the logic which isn't immediately obvious as-is.